app/models/program.py

```diff
@@ -10,6 +10,7 @@ class Program(baseModel):
     isBonnerScholars = BooleanField(default=False)
     contactName = CharField(null=True,default='')
     contactEmail = CharField(null=True,default='')
+    programLocation = CharField(null=True,default='')
```

Comment:
I think this should be `defaultLocation` instead. There isn't really a location for each program, there is a default location that events can override.

Reply:
fixed!